Assertions fail on unicode literals in cli/test_activationkey.py/test_negative_create_with_invalid_name

Currently, our `test_negative_create_with_invalid_name` fails on testing unicode literals, since the string of the error message is being unicode-escaped. We need to encode it back in order to be able to compare it:

> 

E   AssertionError: **u'\u7b7f\u7c4a...'** not found in u'Failed to create ActivationKey with data:\n{\n  "content-view": null, \n  "content-view-id": null, \n  "description": null, \n  "lifecycle-environment": null, \n  "lifecycle-environment-id": null, \n  "max-hosts": null, \n  "name": **"'\\\u7b7f\\\u7c4a'**

--- a/robottelo/test.py
+++ b/robottelo/test.py
@@ -180,7 +180,9 @@ class CLITestCase(TestCase):
         :param contents: contents which must be present on message
         """
         exception = raise_ctx.exception
-        error_msg = getattr(exception, 'stderr', exception.message)
+        error_msg = getattr(exception, 'stderr', exception.message).decode(
+            'unicode-escape'
+        )
         for content in contents:
             self.assertIn(content, error_msg)
 

